src/redux/coinlore/CoinSlice.js

```diff
@@ -14,7 +14,7 @@ export const getCoins = createAsyncThunk('crypto/fetchCrypto', async (_, api) =>
     const res = await axios.get(getCoinUrl);
```

Comment:
- [ ] You have some failing test in `App.js`, kindly fix the failimg test
![image](https://github.com/Simpleshaikh1/Metric-App/assets/81932478/cf4d009b-d337-49c7-879c-c601d108d550)